### ISSUE TITLE
Can't debug Laravel application

### ISSUE THREAD

**Issue by jfbueno** [felixfbecker/vscode-php-debug#179]
I'm working in an application made with Laravel (version 5.4.28) and can't debug my code using this extension.

[StackOverflow has a question](https://stackoverflow.com/q/44669744/4190610) with same problem.  The difference is that, in my case, the exception thrown is a `DecryptException` wich says: 

> `Illuminate\Contracts\Encryption\DecryptException: The payload is invalid.`

It occurs on `Encrypter.php` file from Laravel, [in this line](https://github.com/laravel/framework/blob/v5.4.28/src/Illuminate/Encryption/Encrypter.php#L179).

The most weird thing is that this exception only happens when trying to debug, never when I run the application.

I've already tried to debug using "*launch currently open script*", but it fails with the message

> Fatal error: Class 'MyBaseController' not found

In this case, I'm trying to execute a controller which extends MyBaseController.
 
Something like 

```

class MyController extends MyBaseController {
    function index() {
        dd('just a test');
    }
}

```

---

**PHP version**:  5.6.30
**XDebug version**:  2.5.4
**Adapter version:**  I don't know where I can check this

launch.json content:  
```
{
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Listen for XDebug",
            "type": "php",
            "request": "launch",
            "port": 9000
        },
        {
            "name": "Launch currently open script",
            "type": "php",
            "request": "launch",
            "program": "${file}",
            "cwd": "${fileDirname}",
            "port": 9000
        }
    ]
}

```
XDebug php.ini config:  
Just like the configuration in manual.

**Comment by kajes** [comment]
I think you need to be more specific about how the exception is being thrown. Is it showing in the web browser or in VS Code when you run the debugger?

In the latter case, that would indicate that the debugger indeed is working as expected, but that your PHP error reporting is set to off and not showing any errors when they occur if you don't run the debugger.

You should first check if the error reporting in PHP is on or not. Open your [php.ini](http://php.net/manual/en/function.php-ini-loaded-file.php) file and look for these two lines:
`error_reporting = <Whatever the value is set to>`
and
`display_errors = Off`

These should be set to
`error_reporting = E_ALL`
or
`error_reporting = E_ALL & ~E_DEPRECATED & ~E_STRICT`
and
`display_errors = On`

Then restart the server. If your application is throwing the exception while you run the application without the debugger active, this is not a problem with PHP-Debugger.

**Comment by felixfbecker** [comment]
This looks like working as designed. You can disable breaking on exceptions in the bottom-left section of the debug sidebar.

**Comment by jfbueno** [comment]
@kajes The exception is being thrown in VS Code as soon as I run the debugger.

In php.ini `error_reporting` is set to `E_ALL & ~E_DEPRECATED & ~E_STRICT` and `display_errors` is set to `on`.

I know it's not a properly problem with PHP-Debugger itself, but I also knew that this question would not be well received on StackOverflow, so I decided to try here. 

@felixfbecker Of course it is working. As I already said I came to get some assistance. Probably someone has already gone through this and can help me.
